andela/ah-backend-stark
# #161786659 fix sending of verification link to email

#### What does this PR do?
It fixes the new user email verification feature
#### Description of Task to be completed?

- Create an account with an email service provider
- Configure account smtp credentials in project
- Write functionality to send verification link to email

#### How should this be manually tested?

- Register a new user with a valid email via postman. 
- Log into the registered user's email account.
- You should see an email from author's haven with an account activation link

#### Any background context you want to provide?
The project was initially using sendgrid's API to send signup verification links to email but it stopped working.
#### What are the relevant pivotal tracker stories?
[#161786659](https://www.pivotaltracker.com/story/show/161786659)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/38511070/48156447-7ea28f80-e2de-11e8-8876-3efc75e6a611.png)
